examples/light-client-dapp/src/components/Transfer.tsx

```diff
@@ -2,18 +2,66 @@ import { FormEvent, useCallback, useEffect, useState } from "react"
 import { ss58Decode } from "@polkadot-labs/hdkd-helpers"
 import { UnstableWallet } from "@substrate/unstable-wallet-provider"
 import { toHex } from "@polkadot-api/utils"
+import { filter, firstValueFrom } from "rxjs"
+import { getObservableClient } from "@polkadot-api/client"
+import { ConnectProvider, createClient } from "@polkadot-api/substrate-client"
+import { getDynamicBuilder } from "@polkadot-api/metadata-builders"
+import Select from "react-select"
+
+type SystemAccountStorage = {
+  consumers: number
+  data: {
+    flags: bigint
+    free: bigint
+    frozen: bigint
+    reserved: bigint
+  }
+  nonce: number
+  providers: number
+  sufficients: number
+}
 
 type Props = {
   provider: UnstableWallet.Provider
 }
 
+const getBalance = (provider: ConnectProvider) => async (address: string) => {
```

Comment:
refactor into a hook, something like `useSystemAccount(provider, address)`

It would be nice to model the balance as an observable, so when the transfer is completed the balance is automatically updated